scoped_model_counter/README.md

```diff
@@ -0,0 +1,26 @@
+# ScopedModel Counter
+
+The starter Flutter application, but using ScopedModel to manage state.
+
+This app is a direct counterpart to the [simple counter application][] that you get when you create
```

Comment:
We've done all the other URLs in these files as inlines, which I'd prefer to maintain for consistency's sake if nothing else.